core/src/main/scala/kafka/coordinator/transaction/TransactionCoordinator.scala

```diff
@@ -423,7 +424,7 @@ class TransactionCoordinator(brokerId: Int,
   def partitionFor(transactionalId: String): Int = txnManager.partitionFor(transactionalId)
 
   private def expireTransactions(): Unit = {
-    txnManager.transactionsToExpire().foreach { txnIdAndPidEpoch =>
+    txnManager.timedOutTransactions().foreach { txnIdAndPidEpoch =>
```

Comment:
nit: change the scehduled task name to `scheduler.schedule("transaction-abort"...)` as well?